docs/src/app/shared/markdown-viewer/markdown-viewer.component.ts

```diff
@@ -11,29 +11,20 @@ import { moduleParams } from '../module-params';
   selector: 'sbb-markdown-viewer',
   template: '',
   styleUrls: ['./markdown-viewer.component.scss'],
-  standalone: false,
+  host: {
+    '[innerHTML]': 'content()',
+  },
 })
-export class MarkdownViewerComponent implements OnDestroy {
-  @HostBinding('innerHTML')
-  content!: SafeHtml;
+export class MarkdownViewerComponent {
+  private _htmlLoader = inject(HtmlLoader);
+  private _route = inject(ActivatedRoute);
+  private _domSanitizer = inject(DomSanitizer);
 
-  private readonly _destroyed = new Subject<void>();
-
-  constructor(
-    private _htmlLoader: HtmlLoader,
-    private _route: ActivatedRoute,
-    private _domSanitizer: DomSanitizer,
-  ) {
-    moduleParams(this._route)
-      .pipe(
-        switchMap((params) => this._htmlLoader.withParams(params).fromDocumentation().load()),
-        takeUntil(this._destroyed),
-      )
-      .subscribe((content) => (this.content = this._domSanitizer.bypassSecurityTrustHtml(content)));
-  }
-
-  ngOnDestroy(): void {
-    this._destroyed.next();
-    this._destroyed.complete();
-  }
+  content: Signal<SafeHtml | undefined> = toSignal(
+    moduleParams(this._route).pipe(
+      switchMap((params) => this._htmlLoader.withParams(params).fromDocumentation().load()),
+      map((content) => this._domSanitizer.bypassSecurityTrustHtml(content)),
+      takeUntilDestroyed(),
```

Comment:
I think takeUntilDestroyed() is not needed when wrapped with toSignal() (late finding during my own migrations :D)